scripts/install-mender.sh

```diff
@@ -266,12 +281,48 @@ do_install_commercial() {
     echo "  Success!"
 }
 
+do_setup_mender() {
+    # Return if mender-client was not installed
+    if [[ ! "$SELECTED_COMPONENTS" == *"mender-client"* ]]; then
+        return
+    fi
+
+    # Return if no setup options were passed
+    if [ -z "$MENDER_SETUP_ARGS" ]; then
+        return
+    fi
+
+    echo "  Setting up mender with options: $MENDER_SETUP_ARGS"
+    mender setup $MENDER_SETUP_ARGS
```

Comment:
Suggestion:
Quote the `SETUP_ARGS` here

Reply:
I'll do, thanks.

Reply:
I had to revert this, when `MENDER_SETUP_ARGS` they are passed a single argument to `mender setup`, resulting in the error: `flag provided but not defined: -demo --device-type cool-device --hosted-mender --tenant-token my-secret-token`